app/ui/legacy/src/main/java/com/fsck/k9/ui/settings/AboutFragment.kt

```diff
@@ -142,6 +148,24 @@ class AboutFragment : Fragment() {
                 "https://github.com/splitwise/TokenAutoComplete/",
                 "Apache License, Version 2.0",
             ),
+            Library("accompanist","https://github.com/google/accompanist","Apache License, Version 2.0"),
+            Library("FlexboxLayout","https://github.com/google/flexbox-layout","Apache License, Version 2.0"),
+            Library("Error Prone","https://github.com/google/error-prone","Apache License, Version 2.0"),
```

Comment:
This should be "Error Prone annotations" since that's the only part of Error Prone that is included in the app.